src/amo/css/inc/mixins.scss

```diff
@@ -17,7 +17,14 @@
 @mixin font-family-inter() {
   font-family: 'Inter', sans-serif;
```

Comment:
Is this only important for the variable font file?

Reply:
I didn't want to introduce too much complexity. The intersection of browsers that don't support variable fonts and support `font-display` is probably small anyway...